docs/orchestrating-elastic-stack-applications/elasticsearch/snapshots.asciidoc

```diff
@@ -8,6 +8,8 @@ endif::[]
 [id="{p}-{page_id}"]
 = Create automated snapshots
 
+NOTE: Snapshots are important to ensure that your Elasticsearch indices can be recovered in the event of an accidental deletion, or to migrate data across clusters.
```

Comment:
Should we have this note as a header of the "Run Elasticsearch on ECK" section [here](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-elasticsearch-specification.html). I'm asking because my feeling is that if someone is looking at the snapshot documentation then it may mean that this person is _already_ aware that it's an important topic (or maybe because it's too late 🙃 )

Reply:
Thanks @barkbay for being patient. 
Sure. I added this piece into https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-elasticsearch-specification.html too :) 

(Using @thbkrkr 's suggestion above).

<img width="811" alt="image" src="https://github.com/user-attachments/assets/6e19c8ff-45cf-45fd-ae59-408344d0de14">

Reply:
I think it is worth to repeat important things so it makes logical sense to put it in both below pages.
- https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-elasticsearch-specification.html
- https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-snapshots.html

But I will leave the final decision with you in case you feel we should avoid duplicated description. 

Thanks again!

Reply:
👌